### ISSUE TITLE
Merge (Regex) and  Aggregate Function (sun)

### ISSUE THREAD

**Issue by genofire** [influxdata/influxdb#5268]
What i am doing wrong?
(i am using `v0.9.6.1`. do you need more information?)

```
/query?db=test&epoch=ms&p=[REDACTED]&q=SELECT+sum("value")+FROM+MERGE/httpjson_nodes_.*_statistics_clients/
```
Result:
```
{"error":"error parsing query: found /, expected SELECT, DELETE, SHOW, CREATE, DROP, GRANT, REVOKE, ALTER, SET at line 1, char 31"}
```

---

```
/query?db=test&epoch=ms&p=[REDACTED]&q=SELECT+sum("value")+FROM+MERGE(/httpjson_nodes_.*_statistics_clients/)
```
Result:
```
{"error":"error parsing query: found (, expected regex at line 1, char 31"}
```

**Comment by beckettsean** [comment]
@genofire MERGE is exclusive to InfluxDB 0.8 and prior versions. It is undefined in InfluxDB 0.9, including 0.9.6.1.

Please make sure you are referring to the 0.9 documentation when researching queries.

**Comment by genofire** [comment]
Okay, thats explained a lot (it does not stand anywhere that it does not support it - a tip for the next documents).

But back to my problem, how can i group with aggregate function?
(with using `group by` it group the time and not the measurement)

**Comment by beckettsean** [comment]
@genofire we document the changes from InfluxDB 0.8 to 0.9 on this page: https://docs.influxdata.com/influxdb/v0.9/concepts/08_vs_09/. 

It is not possible to merge results across measurements in InfluxDB 0.9. Measurements should be containers of many series, not individual series themselves. Please review the above page and also https://docs.influxdata.com/influxdb/v0.9/concepts/glossary/ for definitions of `measurement`, `series`, etc.

In a nutshell your current 0.9 schema doesn't support what you want to do. You need to use a schema that is appropriate for 0.9 queries.
